enclave-tls/src/lib/tls_wrappers/internal/tls_wrapper.c

```diff
@@ -0,0 +1,7 @@
+#include <internal/tls_wrapper.h>
```

Comment:
Because all internal header files are not defined in the standard paths, so they should be included in "".